### DISCUSSION TITLE
🙋What do I have to do to save the compiled image as a "tar"

### DISCUSSION THREAD

**Discussion by bigbugcc** [docker/setup-buildx-action#324] [Q&A] [answered]
I appreciate that this project builds multi-platform images very quickly, but I would like to export the compiled image as a tarball to make it easier for me to download it and use it on my LAN; something like `docker save` in docker's command.

**Comment by crazy-max** [comment, marked as answer]
You can use the oci exporter: https://docs.docker.com/build/exporters/oci-docker/

```
$ docker buildx build --output type=oci,dest=image.tar .
```
